### ISSUE TITLE
Social distancing vs social isolation

### ISSUE THREAD

**Issue by AdrianListon** [s-andrews/virusbreak#41]
Should we make a distinction? Anything that reduces interaction is essentially social distancing, so there is a lot of nuance from reducing interaction through to complete lock-down. We could make the current one "social isolation" or "social lockdown" and have a weaker one for "social distancing". Or not, depends on how much work it would be.

**Comment by s-andrews** [comment]
It's not hard to add other categories, it's more thinking about what they're going to mean to the calculation.  At the moment the distancing control stops all movement and reduces the chance of passing along an infection by 10X.  We could have a lesser level which still allows movement and only reduces infection by 5X?

At some point the clarity of any additional controls starts to get lost if there are too many options though.

**Comment by AdrianListon** [comment]
Agreed that too many options are not helpful. Maybe drop quarantine by itself? 

Partial movement and 5x less seems like a good compromise

**Comment by s-andrews** [comment]
I've now implemented the graphing and have turned the isolation effect down to 5X.  At 10X it completed stopped the spread of the virus, whereas at 5X it slows but continues.

![without distancing](https://user-images.githubusercontent.com/8157923/77539599-9362d180-6e99-11ea-9cc9-fc4983cb619f.PNG)

![with distancing](https://user-images.githubusercontent.com/8157923/77539611-978eef00-6e99-11ea-9f45-2d854bde7337.PNG)

**Comment by s-andrews** [comment]
Kind of interesting that you can see waves of infection in the distancing sample.

**Comment by AdrianListon** [comment]
I like the graphs, but perhaps it is better to just have carriers, sick and dead on the graphs. At the moment they are always going to be swamped by naive and immune, which is not the interesting part of the graph.

I think I preferred the 10X reduced rate for social distancing.

**Comment by s-andrews** [comment]
I did try the graphs with the immune and uninfected removed too.  The reason I left them in was that you didn't really get any visual difference in the progress of the infection with and without distancing.  The numbers on the axes are quite different, but the shapes are the same so it's really hard to discern the effect that the intervention had when you don't have any kind of fixed frame of reference.

I guess this will vary by virus.  The corona graph with and without distancing is actually pretty nice with the current setup, but others won't be.

The other thing is that you can rescale the axes.  If you click and drag within the graph area you can expand part of the graph to look at it in more detail.

**Comment by AdrianListon** [comment]
Thinking on this more, I really do prefer the 10X reduced virulence rate for social distancing. It reduces the replication co-efficient to below 1, which fits the real world data.

**Comment by s-andrews** [comment]
OK, I'll put it back then.  It just made it a bit harder to show the curve flattening when the virus died out pretty quickly after starting it.

I've also got some changes to check in around the effect of running out of acute care beds, but I'm not sure I've got the balance of the paramters right yet for a sensible effect.  The simplest case showed no effect of distancing (but it might at 10X reduction), and the more realistic version basically killed everyone!  I've got some compromise code which seems to do better but could do with some testing.
